api/tests/routes/backoffice/offers_test.py

```diff
@@ -1783,6 +1783,39 @@ def test_list_offers_filter_on_offerer(self, authenticated_client):
 
         assert html_parser.count_table_rows(response.data) == 0
 
+    def test_list_offers_on_show_type(self, authenticated_client):
+        offer_to_display = offers_factories.OfferFactory(name="display")
+        offers_factories.OfferFactory(name="hide")
+
+        query_args = {
+            "algolia_search": "display",
+            "limit": 1000,
+            "search-0-search_field": "SHOW_TYPE",
+            "search-0-operator": "IN",
+            "search-0-show_type": ["Cirque"],
+        }
+
+        with patch(
+            "pcapi.routes.backoffice.offers.blueprint.search_offer_ids", side_effect=((offer_to_display.id,),)
```

Comment:
A priori pas besoin de `side_effect` : 
```suggestion
            "pcapi.routes.backoffice.offers.blueprint.search_offer_ids", return_value=(offer_to_display.id,)
```